apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -396,26 +392,24 @@ class ShareAllowedDialog extends React.Component {
                 </div>
                 <div className="social-buttons" style={{marginTop: 12}}>
                   <Button
-                    color={Button.ButtonColor.neutralDark}
-                    id="sharing-phone"
```

Comment:
Any idea what this ID was doing? [This looks maybe relevant](https://github.com/code-dot-org/code-dot-org/blob/fc1a5df28458eb97241a6e953559609a2e27a75f/apps/src/feedback.js#L995)?

Reply:
Yep, also might be the reason of faliing ui test, thanks a lot for spotting!